owners/test/owners_bot.test.js

```diff
@@ -160,6 +170,7 @@ describe('owners bot', () => {
       sandbox.stub(GitHub.prototype, 'createCheckRun');
       sandbox.stub(GitHub.prototype, 'getReviews').returns([]);
       sandbox.stub(GitHub.prototype, 'listFiles').returns([]);
+      sandbox.stub(GitHub.prototype, 'getReviewRequests').returns([]);
```

Comment:
Can we add a use case where `pendingReviewers` exist?

Reply:
There already is one, which directly stubs `_getReviewRequests`. This just blocks all the other tests from making the network request

Reply:
That is tested within OwnersCheck (which is what receives the map of reviewer approvals)

Reply:
Also relevant, I'm planning on splitting the actual running of the owners check out from creating the notifications and adding reviewers, so the notifs/review requests can be triggered independently by #addowners comments. When that happens, it will come along with more tests around how existing reviewers are used. But right now, since the Owners Check is the only consumer of the reviewer statuses, any tests that could be added here would be duplicating tests already in owners check.